### ISSUE TITLE
Interactive Commands cant handle vanilla /fill command

### ISSUE THREAD

**Issue by Nunchakin** [Zrips/CMI#8900]
### Description of Issue

Interactive Commands cant handle vanilla /fill command, it doesnt work and it gives an error in console.

### Version Information

```shell
[12:12:01 INFO]: CMI plugin version: 9.7.4.3 SqLite
>....
[12:12:01 INFO]: CMILib: 1.5.0.5
>....
[12:12:01 INFO]: Server: Paper(44) 1.21-R0.1-SNAPSHOT-
>....
[12:12:01 INFO]: CMI economy: Zapnuté Vault: 1.7.3-CMI CMI Chat: Zapnuté
>....
[12:12:01 INFO]: Modules -> 60 enabled 6 disabled: flightCharge, hpBossBar, chatBubble, tablist, votifier, spawners
```

### Errors

```shell
https://pastebin.com/ApuFtYcQ
```

### Relevant Plugins

Vault version 1.7.3-CMI

### Agreements

- [X] My server is supported by CMI.
- [X] My version of CMI at the time of this report is up to date.
- [X] I have searched the github and asked around before making this report.

**Comment by mrfloris** [comment]
Try adding asPlayer! in front of it? 

asPlayer! fill etcetc

**Comment by Nunchakin** [comment]
Yeah it works, but I dont know how to give players permission for /fill

> Try adding asPlayer! in front of it?
> 
> asPlayer! fill etcetc

**Comment by mrfloris** [comment]
I would never give a player access to fill, so I don't know. Not a CMI permission. Maybe check bukkit perms, or default mc perms, ..
